api/src/rest.rs

```diff
@@ -214,13 +218,26 @@ impl ApiServer {
 			.name("apis".to_string())
 			.spawn(move || {
 				let server = async move {
-					let mut listener = TcpListener::bind(&addr).await.expect("failed to bind");
-					let listener = listener
-						.incoming()
-						.and_then(move |s| acceptor.accept(s))
-						.filter(|r| r.is_ok());
+					let listener = TcpListener::bind(&addr).await.expect("failed to bind");
+
+					let tls_stream = async_stream::stream! {
+						loop {
```

Comment:
Why does this need a loop that was not needed before?

Reply:
Without the loop it would handle a single connection before stopping, this is the common pattern in this particular version of hyper (and beyond, I think, though we'll get to updating hyper to the latest next. An example here https://github.com/hs-CN/async-rustls-stream/blob/ec4298493353d3fae6f86e106d07543b9f1bea56/examples/server_client.rs#L38